### DISCUSSION TITLE
Kebab-case CSS Module convention

### DISCUSSION THREAD

**Discussion by tr1s** [vercel/next.js#12816] [Help]
I thought my HMR wasn't working when I couldn't get my styles to apply!

If I have `<h1 className={styles.postLink}>Post</h1>` in a Gatsby site with CSS Modules, I can then just call `.post-link` in the CSS/SCSS file instead of `.postLink`. This is a much preferred convention when dealing with CSS. 

I presumed it might have been a CSS Modules default _(to allow kebab-case)_, seeing as how there's no mention of it in the Gatsby docs.

**Is there a reason why Next.js doesn't allow it?**

Is it possible to add it in yourself with some configuration,  and/or will that option be added in the future? (presuming it wasn't purposely removed)

**Comment by Tylerian** [comment]
I just want to point out that you can call the style between brackets (i.e: `styles['post-link']`) as a workaround. You lose type safety in favour of keeping your CSS conventions 😅.

**Reply by tr1s** [reply]
Interesting, thanks! Can you tell me more about type safety? Not familiar.
